c/polyjuice.h

```diff
@@ -703,14 +704,14 @@ int load_globals(gw_context_t* ctx, uint32_t to_id, evmc_call_kind call_kind) {
 
   uint8_t creator_script_buffer[GW_MAX_SCRIPT_SIZE];
   mol_seg_t creator_script_seg;
-  mol_seg_t *creator_raw_args_seg_ptr = NULL;
+  mol_seg_t creator_raw_args_seg_ptr;
```

Comment:
Remove `_ptr` suffix, since it's not a pointer.

Reply:
b60f408